test/__tests__/patterns.js

```diff
@@ -0,0 +1,72 @@
+import { runCli } from "../utils";
+
+// MIGRATION: Worked as is.
+describe("multiple patterns", () => {
+  runCli("patterns", ["directory/**/*.js", "other-directory/**/*.js", "-l"]).test({
+    status: 1,
+  });
+});
+
+// MIGRATION: Error is no longer reported when there are other matches, even with
+// `--error-on-unmatched-pattern`. Adjusted the stderr snapshot for now.
+// Should we change the logic to report the invalid patterns regardless?
+// Changed tha status to 1.
+describe("multiple patterns with non exists pattern", () => {
+  runCli("patterns", ["directory/**/*.js", "non-existent.js", "-l"]).test({
+    status: 1,
+  });
+});
+
+// describe("multiple patterns with ignore nested directories pattern", () => {
+//   runCli("patterns", ["**/*.js", "!**/nested-directory/**", "-l"]).test({
+//     status: 1,
+//   });
+// });
+
+// describe("multiple patterns by with ignore pattern, ignores node_modules by default", () => {
+//   runCli("patterns", ["**/*.js", "!directory/**", "-l"]).test({
+//     status: 1,
+//   });
+// });
+
+// describe("multiple patterns by with ignore pattern, ignores node_modules by with ./**/*.js", () => {
+//   runCli("patterns", ["./**/*.js", "!./directory/**", "-l"]).test({
+//     status: 1,
+//   });
+// });
+
+// describe("multiple patterns by with ignore pattern, doesn't ignore node_modules with --with-node-modules flag", () => {
+//   runCli("patterns", ["**/*.js", "!directory/**", "-l", "--with-node-modules"]).test({
+//     status: 1,
+//   });
+// });
+
+// describe("no errors on empty patterns", () => {
+//   // --parser is mandatory if no filepath is passed
+//   runCli("patterns", ["--parser", "babel"]).test({
+//     status: 0,
+//   });
+// });
+
+// MIGRATION: Updated the snapshot and changed the status to 1.
```

Comment:
What was the original status, 2?

Reply:
Yup.

Reply:
It's fine changing it to 1, we don't return 2 ever, and the old CLI was returning 2 or 1 arguably inconsistently, I don't think that was that useful.

Reply:
For sure! I believe you said it was fine in the issue, just wanted to document what changed here compared to the old implementation.